api/apps/api/src/modules/scenarios/blm-calibration/change-scenario-blm-range.command.ts

```diff
@@ -0,0 +1,31 @@
+import { Command } from '@nestjs-architects/typed-cqrs';
+import { Either } from 'fp-ts/Either';
+import { Blm } from '@marxan-api/modules/blm';
+import { GetScenarioFailure } from '@marxan-api/modules/blm/values/blm-repos';
+
+export const invalidRange = Symbol(`invalid range`);
+export const unknownError = Symbol(`unknown error`);
+export const updateFailure = Symbol(`update failure`);
+export const planningUnitAreaNotFound = Symbol(
+  `could not query planning unit area for project`,
+);
+
+export type PlanningUnitAreaNotFoundError = typeof planningUnitAreaNotFound;
+
+export type ChangeScenarioRangeErrors =
+  | typeof invalidRange
+  | typeof unknownError
+  | typeof updateFailure
+  | PlanningUnitAreaNotFoundError
```

Comment:
Why it is possible to return `PlanningUnitAreaNotFoundError` while trying to update scenario blm range? Reviewing the code, it is handled in `ProjectControllers` but I've not seen any function returning this error with `left()`. Note that it is also included in `api/apps/api/src/modules/projects/blm/change-project-blm-range.command.ts`